docs/src/api/class-websocketroute.md

```diff
@@ -0,0 +1,166 @@
+# class: WebSocketRoute
+* since: v1.48
+
+Whenever a WebSocket route is set up with [`method: Page.routeWebSocket`] or [`method: BrowserContext.routeWebSocket`], the `WebSocketRoute` object allows to handle the WebSocket.
+
+By default, the routed WebSocket will not actually connect to the server. This way, you can mock entire communcation over the WebSocket. Here is an example that responds to a `"query"` with a `"result"`.
```

Comment:
I think WebSockets are mostly based on JSON, maybe we should give them a JSON.parse and JSON.stringify example here.

Reply:
I'll follow up with better examples.